Gems/PhysX/Code/Editor/EditorJointConfiguration.cpp

```diff
@@ -411,13 +412,14 @@ namespace PhysX
             m_leadEntity);
         if (entity)
         {
+            const bool leadEntityHasRigidActor =
```

Comment:
This will fail to compile in release because `leadEntityHasRigidActor` wont' be used, please add `[[maybe_unused]]`.

Reply:
nice, good point, thanks!